qemu/tests/cfg/rh_kselftests_vm.cfg

```diff
@@ -0,0 +1,11 @@
+- rh_kselftests_vm:
+    only RHEL
+    type = rh_kselftests_vm
+    virt_test_type = qemu
+    kernel_path = "/tmp/kernel"
+    kselftests_path = "/usr/libexec/kselftests"
+    variants:
+        - mm:
+            no s390x
```

Comment:
@mcasquer you only have one variant here and no s390x is selected, can I ask why?